thirdeye/thirdeye-frontend/app/pods/sandbox/controller.js

```diff
@@ -0,0 +1,136 @@
+import Ember from 'ember';
+import moment from 'moment';
+import fetch from 'fetch';
+
+const generateRandomData = (num, min=0, max=100) => {
```

Comment:
You can define this function inside the controller as opposed to making it a global function, in case it gets overwritten accidentally or called in another file. I would suggest doing the same for other instances.

Reply:
deleted